distribution/docs/src/main/resources/content/_whitelists/security-whitelist.adoc

```diff
@@ -6,53 +6,62 @@
 :order: 04
 :summary: ${ddf-security} whitelist.
 
-The following packages have been exported by the ${ddf-security} application and are approved for use by third parties:
-
```

Comment:
❓ Some of the security implementation classes (e.g., `SubjectImpl`, `SecurityManagerImpl`, etc.) are mentioned in our docs and have been since DDF 2.2. Should we open tickets to review that documentation and take those out since they are implementation details?

Reply:
determined that we are not including implementations just because they are referenced in the docs